infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -30,10 +30,20 @@ You could set the environment variables for variables to stored terraform backen
 `export TF_VAR_storage_account_name="foobar"`
 
 ### Using tfvars
-1. Create a copy of the file backend.tfvars.template by running
-`cp backend.tfvars.template backend.tfvars`.
+1. Create a copy of the file `backend.tfvars.template` into your environment directory by running
+`cp backend.tfvars.template ./environments/<environment-name>/backend.tfvars`.
 2. Edit the backend.tfvars to configure the necessary TF backend state.
 
+## Docker setup
+
+To be able to build the Tables and Housetables service images, you will need to ensure Docker is installed and running on your local machine. You can check for this by running a command like `docker info` in the terminal.
+
+Sometimes, the images may stop building or uploading to the ACR, a process that should be completed automatically after the `terraform apply` commands. If a docker related error is shown, it is likely because docker has run out of local memory. To fix this, run `docker system prune --volumes` locally.
+
+### Building OpenHouse project
+
+Before running `terraform apply`, run `./gradlew clean build` from the root directory to configure the JAR files for image builds.
```

Comment:
```suggestion
## Docker Images - Build

Follow the steps described in [SETUP.md](SETUP.md#build-containers) to build docker container images locally for OpenHouse services. Currently, running only tables and housetables service is supported in Azure.

```

Reply:
I am not sure this is fully accurate. All the user needs to do here is run `./gradlew clean build`. The `terraform apply` builds the images and publishes them to the ACR.

Reply:
Got it, that is cool. I updated my recommendation. I didn't fully follow what was supported in container.